brave/brave-ios
# Don't classify ads if restoring a tab or Cache-Control contains no-store

fixes https://github.com/brave/brave-rewards-ios/issues/153
fixes https://github.com/brave/brave-rewards-ios/issues/158

## Submitter Checklist:


## Test Plan:
- Prefix: enable rewards by removing NO_REWARDS
- Open multiple tabs and load multiple pages and kill app
- Relaunch → ads should not be classified (no report page load for ads lib)
- Load a website that's response header Cache-Control contains no-store → ads should not be classified (no report page load for ads lib)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
